websockets/src/operations.ts

```diff
@@ -0,0 +1,80 @@
+import { HandlerContext, TransactionContext, Transaction, GetApi, ArgSource, ArgSources } from '@dbos-inc/dbos-sdk';
+import { Knex } from 'knex';
+import { WebSocket } from 'ws';
+
+
+// The schema of the database table used in this example.
+export interface dbos_hello {
+  name: string;
+  greet_count: number;
+}
+
+export class Hello {
+
+  @GetApi('/') // Serve a quick readme for the app
+  static async readme(_ctxt: HandlerContext) {
+    const readme = `<html><body><p>
+           Welcome to the DBOS Hello App!<br><br>
+           Visit the route /greeting/:name to be greeted!<br>
+           For example, visit <a href="/greeting/dbos">/greeting/dbos</a>.<br>
+           The counter increments with each page visit.<br>
+           If you visit a new name like <a href="/greeting/alice">/greeting/alice</a>, the counter starts at 1.
+           </p></body></html>`;
+    return Promise.resolve(readme);
+  }
```

Comment:
We should either remove this, or customize it for the application

Reply:
The essence of this app is -- that you start with a npx create .... generated app and then add websockets support to it.

That way the user can create the same on their own.

That is why I prefer not to delete any generated content or comments. If we don'nt like the comments, let us fix it in npx create ...

The existing methods are also useful because it is easy to get the websocket uri wrong and having the original http urls will let the user test that the app is correctly deployed and running

Reply:
I think our demo apps should be targeted to a single purpose. This is otherwise confusing for the user.

The admin API will always be available for testing the app is deployed correctly, so you don't need custom endpoints.

Reply:
I actually think this should not be removed, but changed to reflect what the socket app does. This is usually where a new user goes first after they deploy the app.

Reply:
+1 for customizing the readme for what this demo app demonstrates